docs/components/integration-framework/connectors/out-of-the-box-connectors/aws-sqs.md

```diff
@@ -0,0 +1,71 @@
+---
+id: aws-sqs
+title: AWS Simple Queue Service Connector
+description: Send messages to AWS Simple Queue Service from your BPMN process.
+---
+
+The AWS SQS Connector allows you to connect your BPMN service with the Amazon Web Service's [Simple Queue Service](https://aws.amazon.com/sqs/).
```

Comment:
Minor tweak. Update text to: "The AWS SQS Connector allows you to connect your BPMN service with Amazon Web Service's [Simple Queue Service](https://aws.amazon.com/sqs/)."